OracleWebCenterContent/dockerfiles/README.MD

```diff
@@ -0,0 +1,315 @@
+Oracle WebCenter Content on Docker
+==================================
+
+## Contents
+
+### 1. [Introduction](#1-introduction-1)
+### 2. [Hardware and Software Requirements](#2-hardware-and-software-requirements-1)
+### 3. [Pre-requisites](#3-pre-requisites-1)
+### 4. [Building Oracle WebCenter Content Docker Image](#4-building-oracle-webcenter-content-docker-image-1)
+### 5. [Running Oracle WebCenter Content Docker Container](#5-running-oracle-webcenter-content-docker-container-1)
+### 6. [License](#6-license-1)
+### 7. [Copyright](#7-copyright-1)
+# 1. Introduction
+This project offers scripts to build an Oracle WebCenter Content docker image based on 12.2.1.4.0 release. Use this Docker configuration to facilitate installation, configuration, and environment setup for DevOps users. For more information about Oracle WebCenter Content, see the [Oracle WebCenter Content 12.2.1.4.0 Online Documentation](https://docs.oracle.com/en/middleware/webcenter/content/12.2.1.4/index.html).
+
+This repository includes quick start dockerfiles for WebCenter Content 12.2.1.4.0 based on Oracle Linux 7, Oracle JRE 8 (Server) and Oracle WebLogic Infrastructure 12.2.1.4.0.
+
+The containers will be connected using a Docker User Defined network.
+More information on Docker and its installation in Oracle Linux can be found here:
+https://community.oracle.com/docs/DOC-914969
```

Comment:
This article is five years old and woefully out of date. Please replace it with links to the actual Oracle Linux documentation, i.e [Oracle Container Runtime for Docker User Guide](https://docs.oracle.com/en/operating-systems/oracle-linux/docker/) for Oracle Linux 7 and [Podman User Guide](https://docs.oracle.com/en/operating-systems/oracle-linux/podman/) for Oracle Linux 8.